.github/workflows/ci.yml

```diff
@@ -47,6 +47,9 @@ jobs:
           _NORMALIZED_ENV_NAME=$(echo $DATA_PROJECT_DIR | rev | cut -d "/" -f 1 | rev | tr '.-' '_')
           GIT_BRANCH=${GITHUB_HEAD_REF}
           cat .tinyenv >> $GITHUB_ENV
+          if [ "$(tail -c1 $GITHUB_ENV | wc -l)" -eq 0 ]; then
+            echo >> $GITHUB_ENV # Add a new line if the file does not end with a new line
+          fi
```

Comment:
Not sure if we could do something simpler:

```
cat .tinyenv >> $GITHUB_ENV
echo >> $GITHUB_ENV
```

And we do it always at the end son we can override any FF in the template with `.tinyenv`

Reply:
Yes, simpler and safer